ec/src/models/mod.rs

```diff
@@ -111,6 +113,24 @@ pub trait SWModelParameters: ModelParameters {
 
         res
     }
+
+    /// Generates a uniformly random affine instance of the group.
+    /// Implements `Distribution<GroupProjective<Self>>` for `Standard`.
+    #[inline]
```

Comment:
I moved the inline directive to the method in the ModelParameters, but removed the ones in each group impl. Not sure if that's acceptable.